jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/LambdaBuilder.kt

```diff
@@ -117,6 +117,15 @@ abstract class LambdaBuilder {
                 }
             }
 
+            samOptions.additionalBuildArgs?.let {
+                if (it.isNotBlank()) {
+                    val argParts = it.split(" ")
```

Comment:
in terms of style here you should be able to pass these as `varargs` with the spread operator, rather than the `for (x in y)` style:

`commandLIne.withParameters(*it.split(" ").toTypedArray())`

alternatively you could do it in functional style:
`it.split(" ").forEach(commandLine::withParameters)`